UPGRADING

```diff
@@ -61,6 +61,9 @@ PHP 8.5 UPGRADE NOTES
     array value reference assignment: $ctor_args = [&$valByRef]
   . Attempting to modify a PDOStatement during a call to PDO::fetch(),
     PDO::fetchObject(), PDO::fetchAll() will now throw an Error.
+  . The value of the constants PDO::FETCH_GROUP, PDO::FETCH_UNIQUE,
```

Comment:
I suppose you changed the values initially to be able to put them in a bitfield. Now that you don't seem to do that (anymore?) it's probably better to keep the old values.